doc/how_to_contribute/how_to_contribute_to_site.rst

```diff
@@ -0,0 +1,73 @@
+How-To Contribute to This Site
+==============================
+
+This explains how to make and submit changes to this site.
+This how-to guide assumes you are on Ubuntu 20.04.
```

Comment:
Would a section for "requirements" or "what you need" be good for a standard structure in the how-tos?

Reply:
I like that.  You could also add that to the tutorials for how to write a how-to?